### PR TITLE
Update Dockerfile and README

### PR DESCRIPTION
**Pull request by leelaylay**
1. Some trial examples need extra python pip package such as pandas and lightgbm. 
2. Modify README in docker and networkmorphism_tuner folder.

### CODE REVIEW

**Comment by xuehui1991** [comment]
Shall we merge to v0.5 first?

File: deployment/docker/README.md [hunk 1]
```diff
@@ -6,11 +6,13 @@ This is the Dockerfile of nni project. It includes serveral popular deep learnin
...
 numpy 1.14.3,scipy 1.1.0
-TensorFlow 1.5.0
+TensorFlow 1.10.0
 Keras 2.1.6
 PyTorch 0.4.1
 scikit-learn 0.20.0
-NNI v0.3
+pandas 0.23.4
+lightgbm 2.2.2
+NNI v0.4.1
```
Comment (yds05) [inline deployment/docker/README.md]: should be changed to 'NNI v0.5' ?

**Comment by yds05** [comment]
Suggest to change the PR's target branch from master to v0.5

File: deployment/docker/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (leelaylay) [reply deployment/docker/README.md]: I will change to NNI v0.5 later.

Comment (leelaylay) [reply deployment/docker/README.md]: Sure.

**Comment by leelaylay** [comment]
> Suggest to change the PR's target branch from master to v0.5

Thank you for your advice.

**Pull request merged**